devlab/tests/functional_test.py

```diff
@@ -182,3 +183,10 @@ def get_vms_with_fip_associated(self):
         vms = config.vms
         [vms.extend(i['vms']) for i in config.tenants if 'vms' in i]
         return [vm['name'] for vm in vms if vm.get('fip')]
+
+    def is_tenant_exist(self, keystone_client, tenant_id):
```

Comment:
We have similiar method in generate_load (https://github.com/MirantisWorkloadMobility/CloudFerry/blob/devel/devlab/tests/generate_load.py#L249)
Please remove one of them.